README.md

```diff
@@ -93,6 +94,23 @@ Finally, install the dependencies:
 npm install
 ```
 
+## Faster development building with ic-design-system
+
+If you want to build the components in this repository and use it in [`ic-design-system`](https://github.com/mi6/ic-design-system), you can use `npm link` in `ic-design-system`. This will speed up development because you do not have to generate a compressed npm package, but rather directly use the package after building it locally.
```

Comment:
```suggestion
If you want to build the components in this repository and use them in [`ic-design-system`](https://github.com/mi6/ic-design-system), you can use `npm link`. This will speed up development because you do not have to generate a compressed npm package, but rather directly use the package after building it locally.
```
Not sure it needs the second "in `ic-design-system`" as it sounds a bit repetitive